README.md

```diff
@@ -101,6 +101,17 @@ hardhat:
 ape plugins install alchemy
 ```
 
+## Remote Anvil Node
```

Comment:
This should mention Hardhat node and not Anvil node.
This whole doc mentions Anvil several times. Replace that with Hardhat